content/docs/command-reference/import-url.md

```diff
@@ -122,6 +123,11 @@ from the external data source.
   default file name: `<file>.dvc`, where `<file>` is the desired file name of
   the imported data (`out`).
 
+- `--no-exec` - create `.dvc` file without actually downloading `url`. E.g. if
+  file or directory already exist it can be used to skip download.
+  `dvc commit <out>.dvc` should be used to calculate URL and data hash, update
+  generated .`dvc` files and save existing data to the DVC cache.
```

Comment:
```suggestion
  the file or directory already exist it can be used to skip download.
  `dvc commit <out>.dvc` should be used to calculate the URL and data hash, update
  the .`dvc` files, and save existing data to the cache.
```

Reply:
Done in 57054514